src/modules/event/entities/event.entity.ts

```diff
@@ -0,0 +1,57 @@
+import * as timers from 'node:timers';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { EventStatus } from '../domain/event-status';
+import { EventType } from '../domain/event-type';
+
+@Entity('events')
+export class EventEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  title: string;
+
+  @Column({ nullable: true })
+  description: string;
```

Comment:
nullable true thì phải khai báo đúng type

```suggestion
  @Column({ nullable: true, type: xxxx })
  description: string | null;
```